tslint.json

```diff
@@ -1,20 +1,51 @@
 {
-  "extends": "tslint-config-airbnb",
-  "rules": {
-    "no-inferrable-types": true,
-    "prefer-template": [true, "allow-single-concat"],
-    "import-name": [true, {
-      "errorSubclass": "ErrorSubclass",
-      "magicString": "MagicString",
-      "pluginError": "PluginError",
-      "typescript": "ts",
-      "vinyl": "Vinyl",
-      "vinylFs": "vinylFS"
-    }]
-  },
-  "linterOptions": {
-    "exclude": [
-      "src/__test__"
-    ]
-  }
+    "extends": ["tslint-config-airbnb", "tslint-config-prettier", "tslint-sonarts"],
+    "rulesDirectory": ["tslint-microsoft-contrib"],
+    "rules": {
+        "no-inferrable-types": false,
```

Comment:
I didn't mean to overstep. Sorry about that! It's not always clear to me how much thought people put into their lint rules (some, a lot, some, not that much).

Had no intention of committing this, hoping you wouldn't see this. This is why we have PR reviews, to discuss, right? :)

My thought process is this:

if you have a function:

`function func(someValue = true) {
func2(someValue);
}

function func2(someValue: string | boolean) {
if(typeof someValue === "boolean") {
// do something
} else {
// do something else
}
}
`

If, at some point, you decide to change the default value for `func` to string `"true"`, it might have underlying efects, that might not be immediately visible. Explicitly having a type set, might set expectancy better (or, at least, it does for me).

I will revert this nonetheless.